docs/quick-start.md

```diff
@@ -0,0 +1,214 @@
+TigerBeetle Client
+  Hit enter after a semicolon to run a command.
+
+Examples:
+  create_accounts id=1 code=10 ledger=700 flags=linked | history,
+                  id=2 code=10 ledger=700;
+  create_transfers id=1 debit_account_id=1 credit_account_id=2 amount=10 ledger=700 code=10;
+  lookup_accounts id=1;
+  lookup_accounts id=1, id=2;
+```
+
+## 5. Create Accounts
+
+In the REPL, create two accounts on the same ledger using the command:
+
+```console
+create_accounts id=1 code=10 ledger=700,
+                id=2 code=10 ledger=700;
+```
+
+```console
+info(message_bus): connected to replica 0
+```
+
+## 6. Create a Transfer
+
+Now create a transfer of `10` (of some amount/currency) between the two
+accounts.
+
+```console
+create_transfers id=1 debit_account_id=1 credit_account_id=2 amount=10 ledger=700 code=10;
+```
+
+Now, the amount of `10` has been credited to account `2` and debited from
+account `1`.
+
+## 7. Look Up Accounts
+
+Let's query TigerBeetle for these two accounts to verify the transfer we made!
+
+```console
+lookup_accounts id=1, id=2;
+```
+
+```json
+{
+  "id": "1",
+  "user_data": "0",
+  "ledger": "700",
+  "code": "10",
+  "flags": [],
+  "debits_pending": "0",
+  "debits_posted": "10",
+  "credits_pending": "0",
+  "credits_posted": "0"
+}
+{
+  "id": "2",
+  "user_data": "0",
+  "ledger": "700",
+  "code": "10",
+  "flags": "",
+  "debits_pending": "0",
+  "debits_posted": "0",
+  "credits_pending": "0",
+  "credits_posted": "10"
+}
+```
+
+And indeed you can see that account `1` has `debits_posted` as `10` and account
+`2` has `credits_posted` as `10`. The `10` amount is fully accounted for!
+
+You can take a look at the [`Accounts` reference](./reference/accounts.md) to
+understand all of the fields on the accounts.
+
+## Optional: Run a Multi-Node Cluster
+
+Up to this point, we have only shown you how to run a single-node TigerBeetle
+cluster. In production, TigerBeetle is intended to be run with
+[6 nodes](./deploy/cluster.md).
+
+Here, we will show you how to run a 3-node cluster (the idea is the same for 6
+nodes):
+
+First, create the data files for each node:
+
+```console
+./tigerbeetle format --cluster=0 --replica=0 --replica-count=3 0_0.tigerbeetle
+./tigerbeetle format --cluster=0 --replica=1 --replica-count=3 0_1.tigerbeetle
+./tigerbeetle format --cluster=0 --replica=2 --replica-count=3 0_2.tigerbeetle
+```
+
+Note that the data file stores which replica in the cluster the file belongs to.
+
+Start each server in a new terminal window:
+
+```console
+./tigerbeetle start --addresses=3000,3001,3002 0_0.tigerbeetle
+```
+
+```console
+./tigerbeetle start --addresses=3000,3001,3002 0_1.tigerbeetle
+```
+
+```console
+./tigerbeetle start --addresses=3000,3001,3002 0_2.tigerbeetle
+```
+
+TigerBeetle uses the `--replica` that's stored in the data file as an index into
+the `--addresses` provided.
+
+You can connect to the REPL as described above try creating accounts and
+transfers in this cluster.
```

Comment:
We _probably_ should explain what `--addresses` are here. Alternatively, we could just write-out `127.0.0.1:3000` etc explictily, so that it is self-explnanatory. I'd probaly go for the latter, as it answers the obvious question of "well, and what if my other node is on a different computer"?